api4/channel.go

```diff
@@ -527,7 +527,13 @@ func getChannelStats(c *Context, w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	stats := model.ChannelStats{ChannelId: c.Params.ChannelId, MemberCount: memberCount, GuestCount: guestCount}
+	pinnedPostCount, err := c.App.GetChannelPinnedPostCount(c.Params.ChannelId)
```

Comment:
Please add test to `TestGetChannelStats` of `api4/channel_test.go` that validates `stats.PinnedPostCount`?

Reply:
@saturninoabril Please check for the changes!

Reply:
One more request and I'm all good - https://github.com/mattermost/mattermost-server/pull/11840#discussion_r315082010